drivers/clock_control/Kconfig.nrf

```
@@ -57,7 +57,7 @@ endchoice
 config CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
 	bool "LF clock calibration"
 	depends on !SOC_SERIES_NRF91X && CLOCK_CONTROL_NRF_K32SRC_RC
-	default y
+	default y if !SOC_NRF5340_CPUAPP
```

Since this is done to prevent both cores calibrating the clock, how about keeping this option enabled by default, as it was so far, when the network core is not enabled?
```suggestion
	default y if !BOARD_ENABLE_CPUNET
```

---

Yeah that makes sense, will update.